Gemfile

```diff
@@ -1,12 +1,14 @@
+# frozen_string_literal: true
+
 source 'https://rubygems.org'
```

Comment:
Last comment: have we gone through each of these gems and made sure that we set "reasonable" (read: opinionated) configs for anything that _has_ config?

Reply:
err, "last comment on this file"